Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -701,12 +704,12 @@ def url_quota_command():
     return entry
 
 
-def get_categories_command(display_url, custom_only=False):
+def get_categories_command(args):
+    display_urls = argToBoolean(args.get('displayURL'))  # urls returned to context data even if set to false
+    custom_only = argToBoolean(args.get('custom_categories_only', False))
```

Comment:
let's conform to the majority of args, whether it is camelCase or snakify

Reply:
I cannot change the name of these arguments as it will break BC